build/.vsts-pr.yml

```diff
@@ -3,7 +3,14 @@
 
 name: $(Date:yyyyMMdd)$(Rev:.r)
 
-trigger: none
+trigger:
+  paths:
+    include:
+    - '*'
+    exclude:
+    - '*.md'
+    - docs
+    - data

```

Comment:
I think maybe we should remove 'data' from here and ci build as there are the template management UTs that should be run if things change in data/Templates